### ISSUE TITLE
Right-click context item to copy username

### ISSUE THREAD

**Issue by perry-mitchell** [buttercup/buttercup#244]
For accessibility purposes, the username should be copy-able from the context menu.

**Comment by perry-mitchell** [comment]
Perhaps it could even be under the same context item?

```
Copy ->
         Username
         Password
         URL (eventually, if set)
```

**Comment by sallar** [comment]
@perry-mitchell when a user click's on the entry item in the entries list (middle column) in addition to delete and move entry, we can show a "Copy >" submenu which can include username, password, url and all other custom fields also. What do you think?
